theme/js/script.js

```diff
@@ -118,3 +118,11 @@ function addActiveClassToNavbar() {
         }
     });
 };
+
+function getJsonData() {
```

Comment:
this code is not part of the status bar.

Reply:
I don't see this code in my current branch